docs/legacy/feature-roles.asciidoc

```diff
@@ -230,7 +230,7 @@ IMPORTANT: {deprecation-notice-config}
 You can configure <<api-key-legacy,API keys>> to authorize requests to APM Server.
 To create an APM Server user with the required privileges for creating and managing API keys:
 
-. Create an **API key role**, called something like `apm_api_key`,
+. Create an **API key role**, called something like `apm_agent_key_role`,
```

Comment:
I'm not sure about this renaming. There's no need for an agent's API Key to have `sourcemap:write`. OTOH there's not really any need for a non-agent API Key to have either `config_agent:read` or `event:write`, so the example of adding all three to one API Key probably isn't the best to begin with.

Reply:
Sorry, this one fell off of my radar. 👍 makes sense—will fix!